accounts-db/src/lib.rs

```diff
@@ -5,7 +5,7 @@
 extern crate lazy_static;
 
 pub mod account_info;
-mod account_locks;
+pub mod account_locks;
```

Comment:
this needs to be `pub` so we can actually use the function outside `accounts-db`